css/at-rules/layer.json

```
@@ -0,0 +1,77 @@
+{
+  "css": {
+    "at-rules": {
+      "layer": {
+        "__compat": {
+          "description": "<code>@layer</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@layer",
+          "spec_url": "https://drafts.csswg.org/css-cascade-5/#layering",
+          "support": {
+            "chrome": {
+              "version_added": "97",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "enable-cascade-layers",
```

```suggestion
                  "name": "#enable-cascade-layers",
```

Can we also copy this flag data over to Chrome Android, Edge, and Opera?